src/Quantum/MP.jl

```diff
@@ -110,6 +111,17 @@ end
 const MPS = MatrixProduct{State}
 const MPO = MatrixProduct{Operator}
 
+tensors(ψ::TensorNetwork{MatrixProduct{P,Infinite}}, args...) where {P<:Plug} =
+    throw(ArgumentError("You need to specify the site for an infinite MatrixProduct$P"))
+
```

Comment:
If it doesn't match any method signature, it's better to let Julia throw a `MethodError`.
```suggestion
```

Reply:
Sorry, I `cherry-picked` the commits in #78 because I thought was necessary, I will try to remove them now.